### PR TITLE
revert to default of `host anomaly rate threshold=0.01`

### PR DESCRIPTION
**Pull request by andrewm4894**
##### Summary

This PR reverts the default for `host anomaly rate threshold` to `0.01` from `0.02`. 

The default of this param is 1% because the default of `dimension anomaly score threshold` is `0.99`. This means that typically around 1% of metrics will randomly (naturally) by flipping between anomalous and normal. This is why when the node anomaly rate is above 1% we consider a node level anomaly.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by shyamvalsan** [comment]
@andrewm4894 Don't you think there should be **some** buffer between 

> _the default of dimension anomaly score threshold is 0.99 (meaning that typically around 1% of metrics will randomly by flipping between anomalous and normal)._ 

and what we flag as an anomaly event?

Won't we run the risk of perpetually being on the edge and some non-anomalies would pop up as anomaly events.

As discussed on the call, as long as the changes to introduce longer history and consensus across more models is implemented in the short term (next few weeks) and it succeeds (as it should in theory) to bring down the background anomaly rate to under 1% I'm OK with reverting the change here - but I think we should consider carefully if 1% is the right default regardless. 

My main concern with an unreliable anomaly event trigger is users will start to ignore it (and they might not be sharing feedback with us so we'd never get to know). I'd rather have false negatives vs false positives here.

**Review by stelfrag — APPROVED** [review]

**Comment by andrewm4894** [comment]
> @andrewm4894 Don't you think there should be **some** buffer between
> 
> > _the default of dimension anomaly score threshold is 0.99 (meaning that typically around 1% of metrics will randomly by flipping between anomalous and normal)._
> 
> and what we flag as an anomaly event?

There is - the anomaly event detection logic needs to have sustained elevated rates - we can have a call so I can explain it to you in more detail. 

I am going to insist this PR gets merged. 

We can discuss your ideas and concerns separately. Maybe what we can do is i will add a 2% node to the `devml` nodes and we can have ~identical nodes one with the defaults and the other with 2% and we can see how it all plays out. This is what should have been done at least before we ever merged the original PR that made the change. I will add a 2% node to the scripts and redeploy the devml nodes today and we can see how it looks - might even do a range of different config options as could actually be handy to have them for ml demos etc.

**Comment by andrewm4894** [comment]
@vkalintiris can you approve when you get a chance

**Review by vkalintiris — APPROVED** [review]

**Pull request merged**
